include/range/v3/range/primitives.hpp

```diff
@@ -25,6 +25,8 @@
 
 #include <range/v3/detail/prologue.hpp>
 
+#include <string>
```

Comment:
The only reference to `basic_string` in this header is guarded by `#if RANGES_CXX_STD <= RANGES_CXX_STD_14`, so presumably this could be guarded similarly:
```suggestion
#if RANGES_CXX_STD <= RANGES_CXX_STD_14
#include <string>
#endif
```

Reply:
You're correct ! I commited this change thank you